From 1306f978cadedcfcca674faf86b8fc47b2791be3 Mon Sep 17 00:00:00 2001
From: Felipe Borges <felipeborges@gnome.org>
Date: Fri, 2 Aug 2019 15:36:25 +0200
Subject: [PATCH] linux-user: fix to handle variably sized SIOCGSTAMP with new
 kernels

Based/backported from commit 6d5d5dde9adb5acb32e6b8e3dfbf47fff0f308d2
---
 linux-user/ioctls.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/linux-user/ioctls.h b/linux-user/ioctls.h
index ae8951625f..74476c99e4 100644
--- a/linux-user/ioctls.h
+++ b/linux-user/ioctls.h
@@ -1,3 +1,5 @@
+#include <linux/sockios.h>
+
      /* emulated ioctl list */
 
      IOCTL(TCGETS, IOC_R, MK_PTR(MK_STRUCT(STRUCT_termios)))
-- 
2.21.0

